<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="WorkflowApplicationUnhandledExceptionEventArgs.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac532346c0781c5b93bdde156ae0e7a68ae9bb7bb7f.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">32346c0781c5b93bdde156ae0e7a68ae9bb7bb7f</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>Provides information about an unhandled exception that occurred in a workflow instance.</source>
          <target state="translated">ワークフロー インスタンスで発生した未処理の例外についての情報を提供します。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>If an exception is thrown by an activity and is unhandled, the default behavior is to terminate the workflow instance.</source>
          <target state="translated">アクティビティからスローされた例外が処理されない場合、既定の動作では、ワークフロー インスタンスが終了します。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>If an <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is present, it can override this default behavior.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが存在する場合、このハンドラーで既定の動作をオーバーライドできます。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>This handler gives the workflow host author an opportunity to provide the appropriate handling, such as custom logging, aborting the workflow, canceling the workflow, or terminating the workflow.</source>
          <target state="translated">このハンドラーがあると、ワークフロー ホストの作成者は、カスタムのログ記録、ワークフローの中止、ワークフローのキャンセル、ワークフローの終了などの適切な処理を実行できます。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>The following example invokes a workflow that throws an exception.</source>
          <target state="translated">次の例では、例外をスローするワークフローを呼び出しています。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>The exception is unhandled by the workflow and the <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is invoked.</source>
          <target state="translated">ワークフローで例外が処理されないため、<ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが呼び出されます。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> are inspected to provide information about the exception, and the workflow is terminated.</source>
          <target state="translated">例外に関する情報を提供するために <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> が調査され、ワークフローは終了します。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>Gets the activity that is the source of the unhandled exception.</source>
          <target state="translated">未処理の例外の原因となったアクティビティを取得します。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>An activity.</source>
          <target state="translated">アクティビティ。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>If an exception is thrown by an activity and is unhandled, the default behavior is to terminate the workflow instance.</source>
          <target state="translated">アクティビティからスローされた例外が処理されない場合、既定の動作では、ワークフロー インスタンスが終了します。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>If an <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is present, it can override this default behavior.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが存在する場合、このハンドラーで既定の動作をオーバーライドできます。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>This handler gives the workflow host author an opportunity to provide the appropriate handling, such as custom logging, aborting the workflow, canceling the workflow, or terminating the workflow.</source>
          <target state="translated">このハンドラーがあると、ワークフロー ホストの作成者は、カスタムのログ記録、ワークフローの中止、ワークフローのキャンセル、ワークフローの終了などの適切な処理を実行できます。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>The following example invokes a workflow that throws an exception.</source>
          <target state="translated">次の例では、例外をスローするワークフローを呼び出しています。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>The exception is unhandled by the workflow and the <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is invoked.</source>
          <target state="translated">ワークフローで例外が処理されないため、<ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが呼び出されます。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSource">
          <source>The <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> are inspected to provide information about the exception, and the workflow is terminated.</source>
          <target state="translated">例外に関する情報を提供するために <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> が調査され、ワークフローは終了します。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>Gets the unique identifier of the activity instance that is the source of the unhandled exception.</source>
          <target state="translated">未処理の例外の原因となったアクティビティ インスタンスの一意の識別子を取得します。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>An identifier of the activity instance that is the source of the unhandled exception.</source>
          <target state="translated">未処理の例外の原因となったアクティビティ インスタンスの識別子。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>If an exception is thrown by an activity and is unhandled, the default behavior is to terminate the workflow instance.</source>
          <target state="translated">アクティビティからスローされた例外が処理されない場合、既定の動作では、ワークフロー インスタンスが終了します。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>If an <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is present, it can override this default behavior.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが存在する場合、このハンドラーで既定の動作をオーバーライドできます。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>This handler gives the workflow host author an opportunity to provide the appropriate handling, such as custom logging, aborting the workflow, canceling the workflow, or terminating the workflow.</source>
          <target state="translated">このハンドラーがあると、ワークフロー ホストの作成者は、カスタムのログ記録、ワークフローの中止、ワークフローのキャンセル、ワークフローの終了などの適切な処理を実行できます。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>The following example invokes a workflow that throws an exception.</source>
          <target state="translated">次の例では、例外をスローするワークフローを呼び出しています。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>The exception is unhandled by the workflow and the <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is invoked.</source>
          <target state="translated">ワークフローで例外が処理されないため、<ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが呼び出されます。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.ExceptionSourceInstanceId">
          <source>The <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> are inspected to provide information about the exception, and the workflow is terminated.</source>
          <target state="translated">例外に関する情報を提供するために <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> が調査され、ワークフローは終了します。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>Gets the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that was unhandled by the workflow instance.</source>
          <target state="translated">ワークフロー インスタンスによって処理されなかった <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> を取得します。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that was unhandled by the workflow instance.</source>
          <target state="translated">ワークフロー インスタンスによって処理されなかった <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>If an exception is thrown by an activity and is unhandled, the default behavior is to terminate the workflow instance.</source>
          <target state="translated">アクティビティからスローされた例外が処理されない場合、既定の動作では、ワークフロー インスタンスが終了します。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>If an <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is present, it can override this default behavior.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが存在する場合、このハンドラーで既定の動作をオーバーライドできます。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>This handler gives the workflow host author an opportunity to provide the appropriate handling, such as custom logging, aborting the workflow, canceling the workflow, or terminating the workflow.</source>
          <target state="translated">このハンドラーがあると、ワークフロー ホストの作成者は、カスタムのログ記録、ワークフローの中止、ワークフローのキャンセル、ワークフローの終了などの適切な処理を実行できます。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>The following example invokes a workflow that throws an exception.</source>
          <target state="translated">次の例では、例外をスローするワークフローを呼び出しています。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>The exception is unhandled by the workflow and the <ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> handler is invoked.</source>
          <target state="translated">ワークフローで例外が処理されないため、<ph id="ph1">&lt;xref:System.Activities.WorkflowApplication.OnUnhandledException%2A&gt;</ph> ハンドラーが呼び出されます。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs.UnhandledException">
          <source>The <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> are inspected to provide information about the exception, and the workflow is terminated.</source>
          <target state="translated">例外に関する情報を提供するために <ph id="ph1">&lt;xref:System.Activities.WorkflowApplicationUnhandledExceptionEventArgs&gt;</ph> が調査され、ワークフローは終了します。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>